scipy/signal/_peak_finding.py

```diff
@@ -1225,10 +1225,10 @@ def find_peaks_cwt(vector, widths, wavelet=None, max_distances=None,
         Minimum length a ridge line needs to be acceptable.
         Default is ``cwt.shape[0] / 4``, ie 1/4-th the number of widths.
     min_snr : float, optional
-        Minimum SNR ratio. Default 1. The signal is the value of
-        the cwt matrix at the shortest length scale (``cwt[0, loc]``), the
-        noise is the `noise_perc`th percentile of datapoints contained within a
-        window of `window_size` around ``cwt[0, loc]``.
+        Minimum SNR ratio. Default 1. The signal is the maximum CWT co-efficient
```

Comment:
pretty minor: `coefficient` no hyphen, right?

Reply:
Changed.

Reply:
Not fixed, you put it back with a bad merge @v0dro. I'll review the rest now and will fix this up when merging.